<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BallsBurger Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="result_textbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>321</width>
      <height>551</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>这里会出现生成出来的文本。</string>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="uwant_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>想要什么？</string>
    </property>
   </widget>
   <widget class="QLabel" name="usee_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>会看到什么？</string>
    </property>
   </widget>
   <widget class="QLabel" name="verb_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>给一个对应的动词：</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="uwant_input">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="usee_input">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="verb_input">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>380</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_btn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>460</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>生成</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_btn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>500</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="about_btn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>540</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>关于</string>
    </property>
   </widget>
   <widget class="QLabel" name="lang_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择语言？</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="zhHK_btn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>211</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Traditional Chinese</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="zhCN_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>211</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Simplifiedl Chinese</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
